Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs

```diff
@@ -2,6 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@lucaasrojas is there a reason the license text changed here?

Reply:
@michael-hawker I didn't realize that the text was duplicated.